specs/plasma.md

```diff
@@ -0,0 +1,188 @@
+# Plasma mode
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [DA Storage](#da-storage)
+- [Input Commitment](#input-commitment)
+- [Data Availability Challenge Contract](#data-availability-challenge-contract)
+  - [Parameters](#parameters)
+- [Derivation](#derivation)
+- [Safety and Finality](#safety-and-finality)
+- [Security Considerations](#security-considerations)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
```

Comment:
Perhaps a link to the plasma paper or vitalik's latest post on plasma exit games would be appropriate to gain a background understanding on what plasma is